Formula/statik.rb

```diff
@@ -0,0 +1,125 @@
+    url "https://pypi.python.org/packages/4f/75/e1bc6e363a2c76f8d7e754c27c437dbe4086414e1d6d2f6b2a3e7846f22b/certifi-2016.9.26.tar.gz"
+    sha256 "8275aef1bbeaf05c53715bfc5d8569bd1e04ca1e8e69608cc52bcaac2604eb19"
+  end
+
+  resource "backports-abc" do
+    url "https://pypi.python.org/packages/68/3c/1317a9113c377d1e33711ca8de1e80afbaf4a3c950dd0edfaf61f9bfe6d8/backports_abc-0.5.tar.gz"
+    sha256 "033be54514a03e255df75c5aee8f9e672f663f93abb723444caec8fe43437bde"
+  end
+
+  resource "tornado" do
+    url "https://pypi.python.org/packages/1e/7c/ea047f7bbd1ff22a7f69fe55e7561040e3e54d6f31da6267ef9748321f98/tornado-4.4.2.tar.gz"
+    sha256 "2898f992f898cd41eeb8d53b6df75495f2f423b6672890aadaf196ea1448edcc"
+  end
+
+  resource "livereload" do
+    url "https://pypi.python.org/packages/ba/71/2660028c74cb3289d4b9fd06632aa277b4edbe0747b7219cd92307fa19ba/livereload-2.5.0.tar.gz"
+    sha256 "bc708b46e22dff243c02e709c636ffeb8a64cdd019c95a215304e6ce183c4859"
+  end
+
+  # markdown requirements
+  resource "markdown" do
+    url "https://pypi.python.org/packages/d4/32/642bd580c577af37b00a1eb59b0eaa996f2d11dfe394f3dd0c7a8a2de81a/Markdown-2.6.7.tar.gz"
+    sha256 "daebf24846efa7ff269cfde8c41a48bb2303920c7b2c7c5e04fa82e6282d05c0"
+  end
+
+  # PyYAML requirements
+  resource "PyYAML" do
+    url "https://pypi.python.org/packages/4a/85/db5a2df477072b2902b0eb892feb37d88ac635d36245a72a6a69b23b383a/PyYAML-3.12.tar.gz"
+    sha256 "592766c6303207a20efc445587778322d7f73b161bd994f227adaa341ba212ab"
+  end
+
+  # SQLAlchemy requirements
+  resource "SQLAlchemy" do
+    url "https://pypi.python.org/packages/ca/ca/c2436fdb7bb75d772d9fa17ba60c4cfded6284eed053a7274b2beb96596a/SQLAlchemy-1.1.4.tar.gz"
+    sha256 "701b57d628b9fa1cfb82f10665e7214d5d2db23251ca6f23b91c5f56fcdbdeb5"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    (testpath/"config.yml").write <<-EOS.undent
+      project-name: Homebrew Test
+      base-path: /
+    EOS
+    (testpath/"models/Post.yml").write <<-EOS.undent
+      title: String
+    EOS
+    (testpath/"data/Post/test-post1.yml").write <<-EOS.undent
+      title: Test post 1
+    EOS
+    (testpath/"data/Post/test-post2.yml").write <<-EOS.undent
+      title: Test post 2
+    EOS
+    (testpath/"views/posts.yml").write <<-EOS.undent
+      path:
+        template: /{{ post.pk }}/
+        for-each:
+          post: session.query(Post).all()
+      template: post
+    EOS
+    (testpath/"views/home.yml").write <<-EOS.undent
+      path: /
+      template: home
+    EOS
+    (testpath/"templates/home.html").write <<-EOS.undent
+      <html>
+      <head><title>Home</title></head>
+      <body>Hello world!</body>
+      </html>
+    EOS
+    (testpath/"templates/post.html").write <<-EOS.undent
+      <html>
+      <head><title>Post</title></head>
+      <body>{{ post.title }}</body>
+      </html>
+    EOS
+    system bin/"statik"
+
+    assert File.exist? testpath/"public/index.html"
```

Comment:
Add a second argument to the asserts to describe what the failing assertion means e.g. `, "index.html was not created!"`